coriolis/providers/replicator.py

```diff
@@ -16,7 +16,7 @@
 
 from coriolis import exception
 from coriolis import utils
-
+from coriolis.exception import NotAuthorized
```

Comment:
You can’t import a class name. As per guidelines, it’s:
from coriolis import exception